test_communication/CMakeLists.txt

```diff
@@ -65,21 +65,27 @@ if(BUILD_TESTING)
     if(with_message_argument)
       # adding test for each message type
       foreach(message_file ${message_files})
+        set(_SKIP_TEST "")
```

Comment:
Since the variable has no local scope why should it be prefixed with an underscore? No other variable in this file is.